controllers/gateway/ratelimit/ratelimit_controller_integration_test.go

```diff
@@ -78,14 +79,88 @@ var _ = AfterSuite(func() {
 })
 var _ = Describe("Rate Limit Controller", func() {
 	var ns *corev1.Namespace
+	var apigateway *apigatewayv1alpha1.APIGateway
 	BeforeEach(func() {
+
+		apigateway = &apigatewayv1alpha1.APIGateway{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "custom-apigateway-",
```

Comment:
This test is incorrect. The APIGateway we care about is called `default`. GenerateName does not make sense here